gorm-tools/src/main/groovy/gorm/tools/utils/GormMetaUtils.groovy

```diff
@@ -250,7 +250,10 @@ class GormMetaUtils {
     static List<PersistentProperty> getPersistentProperties(PersistentEntity domain){
         List<PersistentProperty> result = domain.persistentProperties.collect() // collect copies it
         if(domain.compositeIdentity) result.addAll(domain.compositeIdentity)
```

Comment:
no

Reply:
When domain has composite id, the identity comes up null, am i missing some thing ?

Reply:
Fixed